### PR TITLE
[Docs] Updated docs with more detail on AMI builds

### PR DESCRIPTION
**Pull request by sagepe**
Updates relating to the new AMI build process, hopefully the balance is right between what we discuss in the FMS docs and what we discuss in the Public Builds repo - see that repo's [docs branch](https://github.com/mysociety/public-builds/tree/docs) for corresponding additions there.
[skip changelog]

### CODE REVIEW

File: docs/install/ami.md [hunk 1]
```diff
@@ -62,14 +66,34 @@ interface, you will need to create a username and password for one or
...
 
 This basic installation uses the default cobrand, with a
 (deliberately) rather garish colour scheme.  
 
+### Building your own AMI
+
+You may have specific requirements that mean you want to create your own
+customised AMI so we provide access to the Packer configuration files we
+use to generate our images in our [Public Builds repository](https://github.com/mysociety/public-builds)
+together with some guidance on how you can customised the build process.
```
Comment (dracos) [inline docs/install/ami.md]: s/customised/customise/

File: docs/install/ami.md [hunk 2]
```diff
@@ -62,14 +66,34 @@ interface, you will need to create a username and password for one or
...
+
+#### Integrating with other AWS services
+
+One option that we get asked about is how integrate the AMI with other services
```
Comment (dracos) [inline docs/install/ami.md]: s/how/how to/

File: docs/install/ami.md [hunk 3]
```diff
@@ -62,14 +66,34 @@ interface, you will need to create a username and password for one or
...
+within AWS such as RDS. This is more challenging as we can't know about each
+potential environment, but The Public Builds repository contains a recipe that can
```
Comment (dracos) [inline docs/install/ami.md]: s/The/the

**Comment by dracos** [comment]
This presumably should be merged after v2.4.1 is in and generated (and then this can have the AMI id updated before merging).

**Comment by sagepe** [comment]
Typos fixed and notes added to `updating/ami.md`. You are right, we may as well hold this until after the release to save a further subsequent change with the new AMI details.

**Pull request merged**